cu-calendar.sty

```diff
@@ -146,7 +147,8 @@
 	\edef\cuMONTH{\number #2}%
 	\edef\cuDAY{\number #3}%
 	\edef\cuDOW{\intcalcMod{1+\cu@JDN}{7}}%
-	\edef\cuINDICTION{\intcalcAdd{#1}{15 - 15 * \intcalcDiv{#1+14}{15}}}%
+%	\edef\cuINDICTION{\intcalcAdd{#1}{15 - 15 * \intcalcDiv{#1+14}{15}}}%
+	\edef\cuINDICTION{\intcalcAdd{15}{\intcalcMod{#1 - 312}{-15}}}%
 	\edef\cuDISPLAYDATE{\cu@current@dateformat}%
```

Comment:
I couldn't figure out what you were trying to do, but here's a formula that seems to work. Not sure if it works for negative years.